sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'select2-rails', '~> 3.5.9'
   spec.add_dependency 'json-schema'
   spec.add_dependency 'oauth'
+  spec.add_dependency 'devise', '~> 4.2'
```

Comment:
I'm -0 on specifying this change, which has an effect on all downstream apps, to avoid a deprecation warning in the test suite. How about we go the Blacklight route here? https://github.com/projectblacklight/blacklight/blob/master/spec/spec_helper.rb#L56-L61